CineasteTests/MovieListCellTests.swift

```diff
@@ -51,7 +51,7 @@ class MovieListCellTests: XCTestCase {
         XCTAssertEqual(cell.releaseDate.text, "\(storedMovie.releaseDate?.formatted ?? Date().formatted)")
     }
 
-    func testCellIdentifier() {
+    func testCellIdentifierForMovieListCell() {
         XCTAssertEqual(MovieListCell.identifier, "MovieListCell")
```

Comment:
Do we need tests like this? If we change the name of the segue in the storyboard or in the code the app will crash. And this test may fail while everything is fine.

Reply:
You are right. I removed these kind of tests and added some other tests, because what we really want to test is, if the cell identifier string we set in storyboard equals the cell identifier we set in code and if we actually can dequeue a reusable cell with this identifier.